src/rebar_prv_dialyzer.erl

```diff
@@ -374,9 +375,9 @@ run_plt(State, Plt, Output, Analysis, Files) ->
 
 build_proj_plt(State, Plt, Output, Files) ->
     BasePlt = get_base_plt(State),
-    ?INFO("Updating base plt...", []),
     BaseFiles = base_plt_files(State),
     {BaseWarnings, State1} = update_base_plt(State, BasePlt, Output, BaseFiles),
+
```

Comment:
We should only resolve files for the base plt if we are updating the base plt. If we don't update the base plt to match the resolved files then those may not be the ones in the base plt. Instead we should get read the files that are in the base plt.